uw-frame-components/portal/notifications/partials/notification-bell.html

```diff
@@ -31,7 +31,19 @@
 
   <!-- Priority notifications -->
   <div class="priority-notifications" ng-if='notificationsEnabled && (directiveMode === "priority")' ng-repeat="priority in priorityNotifications | limitTo: 1">
-    <a ng-href="{{priority.actionURL}}" alt="{{priority.actionAlt}}" ng-if='priorityNotifications.length == 1' class="notification-message">{{priority.title}}</a>
+    <div ng-if='priorityNotifications.length == 1' layout="row" layout-align="center center" layout-fill>
+      <a ng-href="{{priority.actionURL}}" alt="{{priority.actionAlt}}" class="notification-message">{{priority.title}}</a>
+      <div layout="row" layout-align="center center" ng-if='priority.actionButtons.length > 0' class="notification-buttons">
```

Comment:
First check for `priority.actionButtons` to avoid null reference unhappiness. (Credit to @vertein for catching this.)

`priority.actionButtons && priority.actionButtons.length`